### ISSUE TITLE
event RSVP without creating an account

### ISSUE THREAD

**Issue by steviepubliclab** [publiclab/plots2#157]
It looks like only people who are PL members can RSVP on events. I'll be having PL new comers who might not be ready to register just to RSVP. Don't want to deter people from coming. Perhaps this makes us look like more closed community? Any way we could just have people submit names instead of their user name on this function?

**Comment by jywarren** [comment]
![screen shot 2018-03-26 at 5 38 04 pm](https://user-images.githubusercontent.com/24359/37934534-7c028dbc-311c-11e8-8f15-717f4632cdc3.png)

**Comment by gauravano** [comment]
@jywarren how about opening a new issue for this one? Thanks!

**Comment by jywarren** [comment]
I'm still not sure how we'd do this... it is easier now with the OAuth
login. Is that as easy as entering your email address? Could we consider
making an "incomplete user record" that can't do anything but receive
emails?

**Comment by gauravano** [comment]
I think we can do that. We can create a new table of temporary users which will just contain the required info like name, email, etc. 

We can consider to the same in main users table but if in future user try to create account on Public Lab then it could get complex?

**Comment by gauravano** [comment]
@jywarren

**Comment by jywarren** [comment]
Hmm, I think we could do this but we would have to search for email users
anyways on signup. So maybe we should store them as users with status=5 or
something? And look them up when people sign up, and use the same uid? What
do you think?

But honestly it seems like a fragile system give you can use oath to signup
in such few steps.

**Comment by gauravano** [comment]
Yes, OAuth really makes signup eassyy. I may be wrong but I think that if anyone is really interested about a event then they can signup?

**Comment by jywarren** [comment]
Yes. Let's ping @publiclab/community-reps about this -- it's a substantial
new system to track just emails, but we could do it if it's a priority. Or
we could rely on the newly much easier Login with Twitter/FB/Google process
and close this issue.
